File: adventofcode-2016/src/main/java/nl/mvdr/adventofcode/adventofcode2016/day14/Part1.java
package nl.mvdr.adventofcode.adventofcode2016.day14;

import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

import nl.mvdr.adventofcode.solver.Solver;

public class Part1 implements Solver {

    private static final Logger LOGGER = LoggerFactory.getLogger(Part1.class);

    @Override
    public String solve(String inputfile) {
        return new OneTimePadSolver(1).solve(inputfile);
    }

    public static void main(String[] args) {
        Part1 instance = new Part1();

        String result = instance.solve("input-day14.txt");

        LOGGER.info(result);
    }
}

File: adventofcode-2025/src/main/kotlin/nl/mvdr/adventofcode/adventofcode2025/day06/Part1.kt
package nl.mvdr.adventofcode.adventofcode2025.day06

import io.github.oshai.kotlinlogging.KotlinLogging
import nl.mvdr.adventofcode.solver.FunctionSolver

private val logger = KotlinLogging.logger{}

fun solvePart1(linesSequence: Sequence<String>): Long {
    return 3L // TODO implement
}

fun main() {
    val result = FunctionSolver(::solvePart1).solve("input-day06.txt")
    logger.info { result }
}
